recipes/flair/flair.patch

```diff
@@ -0,0 +1,8 @@
+--- flair.py	2019-09-04 09:47:46.380932333 +0200
++++ flair2.py	2019-09-04 09:48:36.316757866 +0200
+@@ -1,3 +1,5 @@
++#!/usr/bin/env python
```

Comment:
I don't think it will really matter, but can you change this to `/opt/anaconda1anaconda2anaconda3/bin/python`? Conda looks for that in packages and changes the string to whatever the appropriate path is after installation.

Reply:
@dpryan79 is that a recommendation somewhere that I missed. I think its easier to have it more general that like it is.

Reply:
The conda-forge folks have been recommending it. In this case it won't matter, but this is what conda-build looks for to populate the list of "has-prefix" files to modify post-installation.

Reply:
Jupp, but we do not want to that this gets modified isn't it. `/usr/bin/env python` should get the conda one of activated properly...